backend/src/email/middlewares/email-transactional.middleware.ts

```diff
@@ -196,6 +244,29 @@ export const InitEmailTransactionalMiddleware = (
     }
   }
 
+  async function getCCEmails(emailMessageTransactionalId: string) {
+    const cc = (
+      await EmailMessageTransactionalCc.findAll({
+        where: {
+          emailMessageTransactionalId,
+          ccType: CcType.Cc,
+          errorCode: { [Op.is]: null },
+        },
+      })
+    )?.map((c) => c.email)
+    const bcc = (
+      await EmailMessageTransactionalCc.findAll({
+        where: {
+          emailMessageTransactionalId,
+          ccType: CcType.Bcc,
+          errorCode: { [Op.is]: null },
+        },
+      })
+    )?.map((c) => c.email)
```

Comment:
Combine these 2 into a single query and do a filter to separate them out instead